### PR TITLE
Some code documentation updates.

### PR DESCRIPTION
**Pull request by clemens-tolboom**
## Motivation and Context

Make code little more readable
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by ermiyaeskandary** [comment]
Approved - good job!

[Approved with PullApprove](https://pullapprove.com/ErmiyaEskandary/Slither.io-bot/pull-request/246/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ErmiyaEskandary/Slither.io-bot)

**Comment by Seple** [comment]
LGTM

File: bot.user.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ file, You can obtain one at http://mozilla.org/MPL/2.0/.
 // @match        http://slither.io/
 // @grant        none
 // ==/UserScript==
+
```
Comment (ChadSki) [inline bot.user.js:16]: ?

**Comment by ChadSki** [comment]
:+1: Well done

[Approved with PullApprove](https://pullapprove.com/ErmiyaEskandary/Slither.io-bot/pull-request/246/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ErmiyaEskandary/Slither.io-bot)

**Pull request merged**